packages/fp-behandling/src/fakta/components/person/panelBody/arbeidsforhold/PersonArbeidsforholdDetailForm.jsx

```diff
@@ -84,95 +94,98 @@ export const PersonArbeidsforholdDetailForm = ({
           bredde="XL"
           readOnly
         />
-        <BehandlingFormFieldCleaner
-          formName={PERSON_ARBEIDSFORHOLD_DETAIL_FORM}
-          fieldNames={['fortsettUtenImAktivtArbeidsforhold', 'overstyrtTom']}
+        <RadioGroupField
+          name="brukUendretArbeidsforhold"
+          validate={[required]}
+          direction="vertical"
+          readOnly={readOnly}
         >
-          <RadioGroupField
-            name="brukUendretArbeidsforhold"
-            validate={[required]}
-            direction="vertical"
-            readOnly={readOnly}
+          <RadioOption
+            label={intl.formatMessage({ id: 'PersonArbeidsforholdDetailForm.BrukArbeidsforholdet' })}
```

Comment:
Labels for RadioOption kan lages ved å kun sende inn id:
```suggestion
            label={{ id: 'PersonArbeidsforholdDetailForm.BrukArbeidsforholdet' }}
```